s3config-test.yaml

```diff
@@ -52,6 +52,7 @@ S3_SERVER_CONFIG:                                     # Section for S3 Server
    S3_SERVER_REGION_ENDPOINTS: [s3-us-west-2.seagate.com, s3-us.seagate-test.com, s3-europe.seagate-test.com, s3-asia.seagate-test.com]
    S3_SERVER_DEFAULT_ENDPOINT: s3.seagate-test.com
    S3_SERVER_SHUTDOWN_GRACE_PERIOD: 4                   # S3 server shutdown grace period in seconds, default is 10 seconds.
+   S3_RETRY_AFTER_SEC: 30                               # Retry After in seconds
```

Comment:
rename to S3_SHUTDOWN_AFTER_SEC

Reply:
30 second seems high
If failover takes that much time then its ok
Else it can be reduced